
from mymath2 import *
# print(_x) # error

from mymath2 import _y
print(_y) # ok

